assets/css/main.css

```diff
@@ -80,6 +80,11 @@
   margin-top: 100px
 }
 
+.people-img {
```

Comment:
This isn't resizing the images like it is probably intended to do, it's only being put on the `div` that stores the images, causing the over-flow to break the page.